nautobot/circuits/signals.py

```diff
@@ -1,8 +1,34 @@
+from cacheops import invalidate_obj
+from django.contrib.contenttypes.models import ContentType
+
+from django.db import transaction
+from django.db.models import Q
 from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
 from django.utils import timezone
 
 from .models import Circuit, CircuitTermination
+from nautobot.dcim.models import CablePath
+from nautobot.dcim.signals import create_cablepath
+
+
+def rebuild_paths_circuits(obj):
+    """
+    Rebuild all CablePaths which traverse, begin or end with the specified node.
+    """
+
+    cable_paths = CablePath.objects.filter(
+        Q(path__contains=obj)
+        | Q(destination_type=ContentType.objects.get_for_model(CircuitTermination), destination_id=obj.pk)
```

Comment:
Since you're calling `ContentType.objects.get_for_model(CircuitTermination)` twice here, can you please assign it to a variable and reference that twice instead?